.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I don't particularly mind this but is there a reason for the bump?

Reply:
Yes, see e.g. the warnings at the end of this page: https://github.com/scenic-views/scenic/actions/runs/9181920538

Reply:
Nice, thanks.